### PR TITLE
adding flux compatibility section to the release notes 

### PR DESCRIPTION
**Pull request by enekofb**
**What changed?**
Added flux compatibility version to the release notes 

**Why was this change made?**
two main reasons:
- help our users understand compatibility 
- reinforcing our dependency to flux 

**How did you validate the change?**

Pending

### CODE REVIEW

File: .github/workflows/prepare-release.yaml [hunk 1]
```diff
@@ -96,7 +96,19 @@ jobs:
           branch: ${{ env.BRANCH }}
           base: main
           title: "Updates for ${{ env.GITOPS_VERSION }}"
-          body: |
+          body: |                  
+           ## Flux compatibility
```
Comment (stefanprodan) [inline .github/workflows/prepare-release.yaml:100]: I would move this last

**Review by serboctor — APPROVED** [review]

File: .github/workflows/prepare-release.yaml [hunk 2]
```diff
@@ -96,7 +96,7 @@ jobs:
           branch: ${{ env.BRANCH }}
           base: main
           title: "Updates for ${{ env.GITOPS_VERSION }}"
-          body: |
+          body: |
```
Comment (Himangini) [inline .github/workflows/prepare-release.yaml:99]:
```suggestion
          body: |
```

File: .github/workflows/prepare-release.yaml [hunk 3]
```diff
@@ -111,7 +111,14 @@ jobs:
...
              - Any changes in behaviour
              - Changes requiring upgrade notices or deprecation warning
-
+            
+            ## Flux compatibility  
+             | Flux version | Minimum required |
+             |--------------------|------------------|
+             | `v2.0`            | `>= 2.0.0`   |
+            
+             For Flux migrations to v2.0 see [flux](https://github.com/fluxcd/flux2/releases/tag/v2.0.0) or [weave gitops](https://docs.gitops.weave.works/docs/next/guides/fluxga-upgrade/) documentation.
```
Comment (Himangini) [inline .github/workflows/prepare-release.yaml:120]: The link should be https://docs.gitops.weave.works/docs/guides/fluxga-upgrade/

Comment (enekofb) [reply .github/workflows/prepare-release.yaml]: 😅  updated!

**Review by jpellizzari — APPROVED** [review]

**Comment by enekofb** [comment]
Looking good in this week release 

Screenshot 2023-08-30 at 10 59 05

https://github.com/weaveworks/weave-gitops/pull/3970

**Pull request merged**